config/vendor-api-v1.yml

```
@@ -1077,13 +1076,13 @@ components:
           type: string
           format: date
           description: The date the position began
-          example: 2020-09-10
+          example: "2020-09-13"
```

Suggest making this the first of the month, as it always will be, because we don't collect date.

```suggestion
          example: "2020-09-01"
```